sim.py

```diff
@@ -185,7 +198,12 @@ def main(argv: List[str]) -> int:
     last_captured: Optional[GameFrame] = None
 
     obs = OBS(OBS_FPS)
-    gc = GameCapture(obs.composite_interval_ms / 2)
+    if args.capture_ratio is None:
+        gc = GameCapture(obs.composite_interval_ms / 2)
+    elif args.capture_ratio == 0:
+        gc = GameCapture(0)
+    else:
+        gc = GameCapture(obs.composite_interval_ms / args.capture_ratio)
```

Comment:
I would actually make the default for the argument be `2`, and then you can get rid of the special case for `None` -- since really, there's not actually a special case there, not specifying this argument means "use the default", and we can just... specify the default.

Reply:
Wasn't sure if there'd be a use for detecting this pref is explicitly set further down the line, but I don't know of a specific use for that, and certainly for now it just makes this if-else needlessly complicated.